info/v1/container.go

```diff
@@ -293,10 +293,24 @@ type CpuCFS struct {
 	ThrottledTime uint64 `json:"throttled_time"`
 }
 
+// Cpu Aggregated scheduler statistics
+type CpuSchedstat struct {
+	// https://www.kernel.org/doc/Documentation/scheduler/sched-stats.txt
+
+	// These numbers are summed up over all processes in a cgroup and can therefore not capture processes after they are gone.
```

Comment:
this is no longer the case, right?

Reply:
You are right. 
Processes spawned and exiting between housekeeping intervals would still be lost, but I guess that is not comment-worthy. Took it out.